Automattic/vip-cli
# Add the postinstall trackEvent back in

## Description
Adds the postinstall step back in that was revered in #441  

## Steps to Test
Test packaged install:
1. Run `npm pack`
2. Run `npm install -g automattic-vip-1.6.2.tgz` and confirm there are no install errors
3. Might be a good idea to confirm that the track event was sent

Test cloned install: **(this currently does not work)**
1. git clone in a new dir and switch to this branch
2. `npm i` should run without errors
